src/File/UploadFile.php

```diff
@@ -6,77 +6,9 @@
 
 namespace Contentful\File;
 
-class UploadFile implements FileInterface
+/**
+ * @deprecated As of version 2.1
```

Comment:
According to [PSR-5](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#75-deprecated) this should look as follows:

```
@deprecated 2.1 Use RemoteUploadFile instead
@see RemoteUploadFile
```